src/components/tabs/tab/tab.component.tsx

```diff
@@ -23,13 +69,17 @@ const Tab = ({
   updateWarnings,
   updateInfos,
   href,
+  // title is destructured purely to NOT spread it as part of rest to the underlying HTML element.
+  // Both this and titleProps are used as part of child.props inside Tabs component
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   title,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   titleProps,
```

Comment:
this can probably not be destructured and then the linter comment isn't needed

Reply:
read the comment a few lines above - I spent a while on this but unfortunately both this and `title` *do* need to be destructured, despite not being directly used in this component. Otherwise they end up spread onto the styled component. Maybe that's OK for `titleProps` (as opposed to `title` where it definitely isn't) but I'd still rather not run the risk we add something to that that's a valid HTML attribute that ends up causing some weird effect.